doctr/models/classification/mobilenet/pytorch.py

```diff
@@ -51,13 +51,20 @@
         "classes": list(VOCABS["french"]),
         "url": "https://doctr-static.mindee.com/models?id=v0.4.1/mobilenet_v3_small_r-1a8a3530.pt&src=0",
     },
-    "mobilenet_v3_small_orientation": {
+    "mobilenet_v3_small_crop_orientation": {
         "mean": (0.694, 0.695, 0.693),
         "std": (0.299, 0.296, 0.301),
         "input_shape": (3, 128, 128),
-        "classes": [0, 90, 180, 270],
+        "classes": [0, -90, 180, 90],
         "url": "https://doctr-static.mindee.com/models?id=v0.4.1/classif_mobilenet_v3_small-24f8ff57.pt&src=0",
     },
+    "mobilenet_v3_small_page_orientation": {
```

Comment:
We'll need to expose it in `zoo.py` no ?

Reply:
Correct later on i added only the additional dict ftm that you have a place to put the url :)

Reply:
Test integration etc then let n #1553